dolphinscheduler-api/src/main/java/org/apache/dolphinscheduler/api/service/impl/LoggerServiceImpl.java

```diff
@@ -227,4 +235,14 @@ private byte[] getLogBytes(TaskInstance taskInstance) {
         return Bytes.concat(head,
                 logClient.getLogBytes(host.getIp(), host.getPort(), taskInstance.getLogPath()));
     }
+
+    private boolean checkPermissionByTaskInstanceId(User loginUser, int taskInstanceId, String permissionKey) {
+        Project project = projectMapper.queryProjectByTaskInstanceId(taskInstanceId);
+        if (project == null) {
+            return false;
+        }
+        Map<String, Object> result =
+                projectService.checkProjectAndAuth(loginUser, project, project.getCode(), permissionKey);
```

Comment:
could you use `checkProjectAndAuthThrowException` instead of `checkProjectAndAuth` to make our code more clear? It is may ask to change our UT as well